autopush/tests/test_websocket.py

```diff
@@ -54,6 +54,7 @@ def setUp(self):
         self.proto.sendClose = self.close_mock = Mock()
         self.proto.transport = self.transport_mock = Mock()
         self.proto.closeHandshakeTimeout = 0
+        self.proto.autoPingInterval = 300
```

Comment:
This value is set by the config already, since we don't fully instantiate the class here with all the settings, this just sets it up as it will be when used outside of the unit test.